mbzirc_custom/mbzirc_naive_3d_scanning_radar/src/Naive3dScanningRadar.cc

```diff
@@ -113,16 +121,46 @@ void Naive3dScanningRadar::OnRadarScan(const ignition::msgs::LaserScan &_msg)
 
     // Start with -1 x angle step so we just need additions onwards
     double curr_azimuth = _msg.angle_min() - _msg.angle_step();
-    for (uint32_t j = 0; j < _msg.count(); ++j)
+    for (uint32_t j = 0; j < _msg.count(); j += subsampleSize)
     {
-      curr_azimuth += _msg.angle_step();
-      double curr_range = _msg.ranges(ranges_before_channel + j);
-
-      if (curr_range < _msg.range_min() || curr_range > _msg.range_max())
+      double azimuth = 0.0;
+      double range = IGN_DBL_INF;
```

Comment:
Prevents some deprecation warnings, we need to actually prune out those constants.
```suggestion
      double range = ignition::math::INF_D;
```

Reply:
updated. [d0cc571](https://github.com/osrf/mbzirc/pull/171/commits/d0cc5710b0812e90d506f96978b9e015e8c9a9ed)